appnexus/client.py

```diff
@@ -84,7 +84,13 @@ def _send(self, send_method, service, data=None, **kwargs):
             logger.debug(' '.join(map(str, (headers, uri, data))))
 
             response = send_method(uri, headers=headers, json=data)
-            response_data = response.json()
+            content_type = response.headers['Content-Type'].split(';')[0]
```

Comment:
Please stay consistent and use double-quotes rather than single-quotes, here and for the whole diff.